R/common-nelson-rules.R

```diff
@@ -0,0 +1,61 @@
+#' @title
+#' Analyze points in time to deterime whether or not Nelson Rule 1 was violated
+#' @description
+#' Search within a data frame to find violations of Nelson Rule 1.
+#' Nelson Rule 1: One point is more than 3 standard deviations from the mean.
+#' @param df A data frame
+#' @param dateCol A string denoting the date column
```

Comment:
Can you expand what the date column is used for? It's just for plotting over time, right? And you need to add the param, `measureCol`.